### ISSUE TITLE
not compatible with latest quasar framework

### ISSUE THREAD

**Issue by antonic901** [quasarframework/quasar-ui-qmediaplayer#375]
**Describe the bug**
I'm not able to install ```@quasar/qmediaplayer@next```. After installing it and running project, I'm getting following error:
```App • ⚠  Extension(@quasar/qmediaplayer): Dependency not found - @quasar/app. Please install it.```

**To Reproduce**
1. type ```quasar ext add @quasar/qmediaplayer@next```
2. run project ```npm run dev```

**Expected behavior**
npm run dev shouldn't throw error

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Browser: Firefox
 - Version: v133.0.3

**Comment by DrYurets** [comment]
Any ideas?

**Comment by LamentWy** [comment]
well , I'm using ``@quasar/qmediaplayer@next `` recently.

 then I found, even u can install it,  ``quasar dev`` still can't pass. 

output will be  ``  App • ⚠️  Extension(@quasar/qmediaplayer): Dependency not found - @quasar/app. Please install it.``

but I got ``"@quasar/app-vite": "^2.1.0"``, and I really don't want ``@quasar/app``.

now, I'm looking at this issue feel hopeless.

**Comment by hawkeye64** [comment]
I'll look into this over the weekend or as time permits.

**Comment by LamentWy** [comment]
@hawkeye64 maybe no need for that, hope I can save some time for u.

After my last comment, I did some research and seems find out why. 

TLDR : It's a version problem, don't use ``@quasar/qmediaplayer@next``,  use this version ``QMediaPlayer v2.0.0-beta.6`` , 
u can see this version's release note say **feat: support for q/app-vite**.

``@quasar/qmediaplayer`` install failed because **quasar** will check dependency ``@quasar/app``  before install

``quasar dev`` failed because the ``@quasar/qmediaplayer@next`` version does't support ``@quasar/app-vite`` yet.

**Comment by hawkeye64** [comment]
Sounds like the docs needs to change
